### ISSUE TITLE
Spanish Create User Test

### ISSUE THREAD

**Issue by eleanorreem** [chaynHQ/bloom-frontend#837]
### Overview
All our testing is currently in english and we want to ensure our other site languages are working. This need came from an incident where a bug happened on all browser languages except for english and the cypress tests did not catch it.

### Action Items

- [ ] Get Cypress tests up and running. See [Cypress Readme](https://github.com/chaynHQ/bloom-frontend/blob/develop/CYPRESS.md)
- [ ] Create your own branch 
- [ ] Take a look at this [german example](https://github.com/chaynHQ/bloom-frontend/blob/develop/cypress/integration/create-user.de.cy.tsx) of the test 
- [ ] Duplicate this test in Spanish and name it create-user.es.cy.tsx. Note that you will have to update some of the test assertions to match the spanish version of the site. Take a look at bloom.chayn.co/es for the spanish version of the site to find the equivalent text. 
- [ ] See notes below for helpers
- [ ] Commit and push your changes and create a PR.
- [ ] Tag @eleanorreem  for a review 

### Resources/Instructions
- [Cypress readme](https://github.com/chaynHQ/bloom-frontend/blob/develop/CYPRESS.md)
- Ensure you create a custom visitSpanishPage that replicates the behaviour of visitGermanPage in [this file](https://github.com/chaynHQ/bloom-frontend/blob/develop/cypress/support/commands.js)
- You will also have to update the [type file](https://github.com/chaynHQ/bloom-frontend/blob/develop/cypress/support/index.d.ts)

**Comment by cmstrandwold** [comment]
Hi! I am interested in working on this issue and am a first-time contributor. Can you assign this to me?

**Comment by kyleecodes** [comment]
Yes you are assigned, thank you for your patience! :)

**Comment by kyleecodes** [comment]
Additionally @cmstrandwold, thank you for your interest in Chayn tech! I just saw your request for the volunteer form data.

This brought to our attention that we had the incorrect link on our Getting Involved page, oh no!
We have fixed this so that Gmail sign-in is no longer required. Our apologies, and thank you for your patience!

Here is the link to the Chayn Tech Volunteer form for your convenience: https://docs.google.com/forms/d/e/1FAIpQLSdAX6B4Om92MUCRLSXR38fJTlAW5E43xIxSe_SIwcN7wru4IA/viewform
